iib/web/static/api_v1.yaml

```diff
@@ -686,6 +700,13 @@ components:
             non-privileged users to use overwrite_from_index. The format of the token must be in
             the format "user:password".
           example: token
+        distribution_scope:
+          description: >
+                The scope of distribution for the index creaded by the request.
+                Will determine what level of protection addition and removal will
+                have.
```

Comment:
```suggestion
                The scope of distribution for the index created by the request.
                This will determine what level of protection removal will have.
```